src/xcode/ENA/ENA/Source/Models/Exposure/Country.swift

```diff
@@ -21,7 +21,7 @@ struct Country: Equatable, Codable {
 		UIImage(named: "flag.\(id.lowercased())")
 	}
 
-	/// Initialize a country with a given `countryCode`. If no valid `countryCode` is given the initalizer returns `nil`.
+	/// Initialize a country with a given `countryCode`. If no valid `countryCode` is given the initializer	 returns `nil`.
```

Comment:
Accidental <kbd>Tab</kbd>?

Reply:
@heinezen

Do you mean at the start or at the end of the line? Cause the tab in at the start was already there and I'm not seeing a tab added. 😅

Reply:
Btw can you add the `Community` label here @heinezen?

Reply:
@Ein-Tim There is a tab right after "initializer": `initializer<TAB>`

You can kind of see it in the Github diff.

Reply:
@heinezen 

Ah, now I know what you mean! Fixed in [`a44dbda` (#3627)](https://github.com/corona-warn-app/cwa-app-ios/pull/3627/commits/a44dbda88f7760d97d58b695e0499227eb2f50cd).